addons/l10n_ch/models/account_bank_statement.py

```diff
@@ -9,7 +9,7 @@ class AccountBankStatementLine(models.Model):
     _inherit = "account.bank.statement.line"
 
     def _find_or_create_bank_account(self):
-        if self.company_id.country_id.code == 'CH' and _is_l10n_ch_postal(self.account_number):
+        if self.company_id.country_id.code in ('CH', 'LI') and _is_l10n_ch_postal(self.account_number):
```

Comment:
will `_is_l10n_ch_postal` pass with a LI address?

Reply:
I did a bit of research now , apparently postfinance (the issuer of postal accounts) also provides services in LI: 
I quote:

> PostFinance offers comprehensive payment transaction solutions for banks and financial service providers based in Switzerland or **Liechtenstein**.

https://www.postfinance.ch/en/business/needs/financial-institutions/switzerland.html

I might be wrong though, can you please confirm? @yvaucher

Reply:
@casp-odoo `_is_l10n_ch_postal` is deprecated, as we only use IBAN since 1st October 2022 for all bank transactions, it is not necessary to consider it.

Reply:
Thus, even for postfinance accounts, the account owner will get an IBAN instead of postal number.